src/scales/scale.time.js

```diff
@@ -37,10 +41,107 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'linear',   // 'linear|series'
+			source: 'auto'    // 'auto|labels'
 		}
 	};
 
+	function sorter(a, b) {
+		return a - b;
+	}
+
+	/**
+	 * Returns an array of {time, pos} objects used to interpolate a specific `time` or position
```

Comment:
Unfortunately not, it's a decimal between [0,1], that's explained [a few lines above](/simonbrunel/Chart.js/blob/dbeb5ea66059e88d772bb0cc8724ef3c2a640007/src/scales/scale.time.js#L59).